CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v1.1.2 - Unreleased
+
+- Update dependencies to latest up to date `gleam_stdlib` and `gleam_json`.
```

Comment:
This would be something like "Relaxed version constraint to permit `gleam_json` v2."